packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -396,73 +396,120 @@ const renderPointGraphControls = (props: {
     );
 };
 
-/**
- * TODO(catjohnson): The polygon controls will be vastly different from the point controls.
- * Will be keeping this function as is for the time being, even if it's a copy of the point graph
- * controls.
- */
 const renderPolygonGraphControls = (props: {
     state: PolygonGraphState;
     dispatch: (action: InteractiveGraphAction) => unknown;
     width: number;
     perseusStrings: PerseusStrings;
 }) => {
-    const {interactionMode, showRemovePointButton, focusedPointIndex} =
-        props.state;
+    const {
+        interactionMode,
+        showRemovePointButton,
+        focusedPointIndex,
+        closedPolygon,
+        coords,
+    } = props.state;
     const {perseusStrings} = props;
 
     const shouldShowRemoveButton =
         showRemovePointButton && focusedPointIndex !== null;
```

Comment:
Since this variable now controls whether the remove button is enabled, I think it should be called `shouldEnableRemoveButton`. It could also incorporate `closedPolygon`:

```suggestion
    const shouldEnableRemoveButton =
        !closedPolygon && showRemovePointButton && focusedPointIndex !== null;
```

I wonder if `state.showRemovePointButton` should also be renamed to `state.enableRemovePointButton`. Or if a different name entirely is appropriate, since that state variable doesn't completely control whether the button is enabled or not.

I'm kind of confused about when the remove button is enabled vs. disabled, and that might be a sign that it will be confusing to learners as well.